packages/k8s/k3s-systemd/collection.yaml

```diff
@@ -0,0 +1,36 @@
+packages:
+  - name: k3s
```

Comment:
name is not changed, there will be two different packages with the same name and category, this need to be changed to k3s-systemd or k3s-openrc

Reply:
yes, saw that now. Fixed.